### ISSUE TITLE
Blender mixdown broken

### ISSUE THREAD

**Issue by karrirasinmaki** [GDquest/BPSRender#8]
**I'm submitting a...**

- [x] Bug report.
- [ ] Feature request.

## Bug report ##

**What is the current behavior?**

Mixdown crashes:
```
~ WARNING:mixdown: Something went wrong when calling command - SKIPPING.
```
The problem probably lies on the Blender side ([bug report filed](https://developer.blender.org/T88614)), since Blender "Render Audio" feature gives this error:
```
terminate called after throwing an instance of 'aud::FileException'
  what():  File couldn't be written, audio encoding failed with ffmpeg.
```

**What is the expected behavior?**

Mixdown should not crash and produce an audio file instead.

**Tell us the steps to reproduce the bug, and if possible share a minimal demo of the problem.** 

Make sure you have Blender 2.92.3 and try rendering a mixdown with bpsrender.

## Other ##

**Other information** 

Bug reported on developer.blender.org:
https://developer.blender.org/T88614

But probably a good idea to have the bug listed here as well if someone is looking at what's wrong.

I tried also with the new Blender 3.0 Alpha and the same error there. With Blender 2.83.15 feature still works, but I'm not sure which version broke it.

**Comment by NathanLovato** [comment]
Thanks but according to your report, it has to do with Blender? If running it from inside Blender fails, then all this script does is asking blender to make a mixdown, so we can't fix it on our side.

**Comment by karrirasinmaki** [comment]
Yep. I just thought maybe it'd be nice to have the bug listed here as well so people can find the cause more easily. At least it took quite a while for me to figure out the root cause maybe lies in Blender itself.
